queries.sql

```diff
@@ -5,4 +5,30 @@ SELECT date_of_birth FROM animals WHERE NAME IN ( 'Agumon','Pikachu');
 SELECT name, escape_attempts FROM animals WHERE (weight_kg > 10.5);
 SELECT * FROM animals WHERE (neutered = true);
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE (weight_kg >= 10.4 AND weight_kg <= 17.3);
\ No newline at end of file
+SELECT * FROM animals WHERE (weight_kg >= 10.4 AND weight_kg <= 17.3);
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE Name like '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT date_deletion;
+UPDATE animals SET weight_kg = weight_kg * (-1);
+UPDATE animals SET weight_kg = weight_kg * (-1) WHERE (weight_kg < 0);
+COMMIT;
```

Comment:
- [ ] Kindly attach new screenshots for these transactions because, in them, you have the entry of the animal `Charmander` twice and based on the requirements, the entry for Charmander should be once. Kindly implement this change. 👍 💻 

![image](https://github.com/KAYEMBAJONATHAN/Vet_Clinic/assets/77106962/668e860e-509b-42dc-a097-1c55f66773a5)